cloudify_cli/replace_certificates_config.py

```diff
@@ -162,18 +163,19 @@ def replace_certificates(self):
         self._handle_new_ca_certs()
         # Passing only the new CA cert to the agents
         self._pass_new_ca_certs_to_agents(bundle=False)
-        self.logger.info('Validating status is healthy')
+        self.logger.info('\nValidating status is healthy')
```

Comment:
Why the newline on a logger line?

Reply:
Just to separate it from the `Replacing certificates on host <host-ip>` logs. As a best-practice, logs shouldn't include newlines.